cpp/src/mip/solve.cu

```diff
@@ -70,6 +70,7 @@ mip_solution_t<i_t, f_t> run_mip(detail::problem_t<i_t, f_t>& problem,
   pdlp_hyper_params::update_step_size_on_initial_solution     = true;
   // if the input problem is empty: early exit
   if (problem.empty) {
+    CUOPT_LOG_INFO("Optimal solution found during presolve");
```

Comment:
This is the wrong place for this message. Problem could be empty for different reason as well.   

It makes sense to move this to presolve logic.